docs/client/initializing.md

```diff
@@ -0,0 +1,83 @@
+# <img align="center" src="../images/logo.png">  Initializing Your Python Client
+
+The first step to using your generated client in code is to import and initialize your client. Our SDKs are modelled such
+that the client is the main point of access to the generated code.
+
+## Importing Your Client
+
+You import your client from the namespace specified when generating (under flag `--namespace`). For the sake of this example,
+let's say the namespace is `azure.pets`. Your client's name is detailed in the swagger, (TODO link to swagger docs), and let's say
+ours is called `PetsClient`.
+
+Putting this together, we import our client like so:
+
+```python
+from azure.pets import PetsClient
+```
+
+## Initializing Your Client
+
+Next, on to initialization. Your constructor can take any number of parameters. If your client has no parameters (no client parameters detailed
+in the swagger (TODO: link to swagger docs) and you choose to generate without credentials), initializing your client would just look like
+
+```python
+from azure.pets import PetsClient
+
+client = PetsClient()
+```
+
+However, by default we generate clients with credentials, so continue on to [Authenticating Your Client](#authenticating-your-client "Authenticating Your Client")
+to find out how to input a credential.
+
+## Authenticating Your Client
+
+By default we generate our clients with an [Azure Active Directory (AAD) token credential][aad_authentication]. We always recommend
```

Comment:
Does it default or do you need the add credentials flag?

Reply:
(this isn't the current behavior, but this is what I think the behavior is going to be): default without specifying you want credentials. We're thinking of deprecating `add-credentials`, and instead encouring new flag `--credential-types`. Default of that being `TokenCredential`, can also specify `AzureKeyCredential` or None